.travis.yml

```diff
@@ -1,8 +1,20 @@
-language: node_js
 sudo: false
+
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
```

Comment:
Does this also mean, that we can't test our PRs in Travis anymore? Because they are not `master` or `greenkeeper/*`.